x/oracle/client/cli/tx.go

```diff
@@ -0,0 +1,172 @@
+package cli
+
+import (
+	"encoding/base64"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdktx "github.com/cosmos/cosmos-sdk/types/tx"
+	"github.com/cosmos/cosmos-sdk/version"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	govcli "github.com/cosmos/cosmos-sdk/x/gov/client/cli"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+
+	"github.com/provenance-io/provenance/x/oracle/types"
+)
+
+// The flag to specify that the command should be ran as a gov proposal
+const FlagGovProposal = "gov-proposal"
```

Comment:
You should only need this flag if there are endpoints that are sometimes used as a normal `Tx`, and sometimes as a governance proposal.